.circleci/config.yml

```diff
@@ -13,8 +13,10 @@ executors:
       CONSUL_ENT_VERSION: 1.11.0+ent-alpha # Consul's enterprise version to use in tests
 
 control-plane-path : &control-plane-path control-plane
```

Comment:
The changes in this file are to 
1) change the directory of the acceptance tests (move them to top level)
2) compile the cli binary into the gopath so it can be used from the acceptance tests for the CLI. 

Note that running the cli acceptance tests requires a local binary on the path called consul-k8s